bundles/org.openhab.binding.benqprojector/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,16 @@
 	<description>This binding is compatible with BenQ projectors</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>sddp</service-type>
+			<match-properties>
+				<match-property>
+					<name>type</name>
+					<regex>(?i)benq:projector.*</regex>
+				</match-property>
```

Comment:
To be consistent with the discovery code, you could have a match for `manufacturer` + a match for `type`.
Take a look, of my proposal for the Sony projector.

Reply:
fixed